3rdParty/fuerte/include/fuerte/message.h

```diff
@@ -226,6 +226,8 @@ class Response final : public Message {
   Response(ResponseHeader&& reqHeader = ResponseHeader())
       : header(std::move(reqHeader)), _payloadOffset(0) {}
 
+  Response(Response const& resp);
```

Comment:
no, just no

Reply:
ok. How to get the reply of an error so proper error messages can be emitted or analyzed?